### PR TITLE
[IMP] website: introduce `s_opening_hours` snippet

### PR DESCRIPTION
**Pull request by xlu-odoo**
This commit adds the new `s_opening_hours` snippet.

task-4140779
Part-of: task-4077427

| New snippet: Opening Hours |
|--------|
|  Capture d’écran 2024-09-05 à 16 39 42  | 

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by anso-odoo — CHANGES_REQUESTED** [review]
Hello @xlu-odoo, first review

There is an issue on tablet layout where the content goes out of the snippet. Maybe you could use `small` text for the days and hours, it may fix your issue
Screenshot 2024-09-04 at 11 41 40

File: addons/website/views/snippets/s_opening_hours.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_opening_hours" name="Opening Hours">
+    <section class="s_opening_hours">
+        <div class="container-fluid">
+            <div class="row o_grid_mode" data-row-count="13">
+                <div class="o_grid_item o_grid_item_image g-col-lg-13 g-height-12 col-lg-12" style="z-index: 1; grid-area: 1 / 1 / 14 / 13; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px;">
```
Comment (anso-odoo) [inline addons/website/views/snippets/s_opening_hours.xml:8]:
```suggestion
                <div class="o_grid_item o_grid_item_image g-col-lg-12 g-height-13 col-lg-12" style="z-index: 1; grid-area: 1 / 1 / 14 / 13; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px;">
```

File: addons/website/views/snippets/s_opening_hours.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                    <img class="img img-fluid" src="/web/image/website.s_cover_default_image" alt=""/>
+                </div>
+                <!-- This next div is intended to correct a visual issue, specifically a vertical line anomaly next to the hours, that appears only in the modal preview.-->
+                <div class="o_grid_item o_cc o_cc1 g-col-lg-10 g-height-7 col-lg-10 s_dialog_preview" style="z-index: 2; grid-area: 7 / 2 / 14 / 12; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px;"/>
```
Comment (anso-odoo) [inline addons/website/views/snippets/s_opening_hours.xml:12]: Nitpicking: our classes always first (same for grid items below)

```suggestion
                <div class="s_dialog_preview o_grid_item o_cc o_cc1 g-col-lg-10 g-height-7 col-lg-10" style="z-index: 2; grid-area: 7 / 2 / 14 / 12; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px;"/>
```

File: addons/website/views/snippets/s_opening_hours.xml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                <div class="o_grid_item g-col-lg-6 g-height-7 col-lg-6 o_cc o_cc1" style="grid-area: 7 / 2 / 14 / 8; z-index: 3; --grid-item-padding-y: 32px; --grid-item-padding-x: 40px;">
+                    <h2 class="display-4-fs" style="text-align: left;">Let’s get in touch</h2>
```
Comment (anso-odoo) [inline addons/website/views/snippets/s_opening_hours.xml:14]: Is there a reason you declare `style="text-align: left;"` here but not on days + hours text ? I guess if it's default, we can get rid of this inline style ? Same for the `lead` paragraph

**Review by anso-odoo — CHANGES_REQUESTED** [review]
Hello, I tested the snippet in different themes and it looks fine 👍 

Some comments for you

**Review by anso-odoo — APPROVED** [review]
LGTM 

----

Note:
https://github.com/odoo/odoo/pull/178097/files#diff-e83e5ba1a55ef7acc864c5547d9515b3302dd8ea869edec1d4cac02eb398ba92R11
For reference, this is the bug we try to fix with this "hack"
Screenshot 2024-09-04 at 11 30 27

**Review by SergeBayet — APPROVED** [review]
LGTM. thanks

**Pull request closed without merging**